doc/user/content/self-hosted/_index.md

```diff
@@ -0,0 +1,262 @@
+```yaml
+storage:
+  storageClass:
+    create: true
+    name: "openebs-lvm-instance-store-ext4"
+    provisioner: "local.csi.openebs.io"
+    parameters:
+      storage: "lvm"
+      fsType: "ext4"
+      volgroup: "instance-store-vg"
+```
+{{< /tab >}}
+{{< tab "Other Storage" >}}
+While OpenEBS is our recommended solution, you can use any storage provisioner that meets your performance requirements by overriding the provisioner and parameters values.
+
+For example, to use a different storage provider:
+
+```yaml
+storage:
+  storageClass:
+    create: true
+    name: "your-storage-class"
+    provisioner: "your.storage.provisioner"
+    parameters:
+      # Parameters specific to your chosen storage provisioner
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+## Installing the Chart
+
+1. If installing for the first time, create a namespace. The default
+   configuration uses the `materialize` namespace.
+
+   ```bash
+   kubectl create namespace materialize
+   ```
+
+1. Install the Materialize operator with the release name
+   `my-materialize-operator`
+
+      - To use the [default configuration](/self-hosted/configuration/):
+
+        ```shell
+        helm install my-materialize-operator materialize/misc/helm-charts/operator
+        ```
+
+      * To include custom configuration, you can:
+
+        - *Recommended:* Create or edita a YAML file (such as the sample
+          `values.yaml` file located in the
+          `materialize/misc/helm-charts/operator/` directory) that specifies the
+          values for the parameters and then install the chart with the `-f`
+          flag:
+
+          ```shell
+          helm install materialize/misc/helm-charts/operator/values.yaml  materialize/misc/helm-charts/operator
+          ```
+
+        - Specify each parameter using the `--set key=value[,key=value]`
+            argument to `helm install`. For example:
+
+            ```shell
+            helm install my-materialize-operator \
+              --set operator.image.tag=v1.0.0 \
+              materialize/materialize-operator
+            ```
+
+1. Verify the installation:
+
+    ```shell
+    kubectl get all -n materialize
+    ```
+
+## Deploying Materialize
+
+### Set up the metadata database
+
+
+### Set up blob storage
+
+
+### Deploy Materialize
+
+To deploy Materialize:
+
+1. Create a YAML file (e.g., `materialize.yaml`) with your Materialize blob
+   storage and metadata database configuration. The following example assumes
+   you have deployed PostgreSQL as the metadata database and MinIO as the blob
+   storage.
+
+
+   ```yaml
+   apiVersion: v1
+   kind: Namespace
+   metadata:
+     name: materialize-environment
+   ---
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: materialize-backend
+     namespace: materialize-environment
+   stringData:
+     metadata_backend_url: "postgres:// materialize_user:materialize_pass@postgres.materialize.svc.cluster. local:5432/materialize_db?sslmode=disable"
```

Comment:
Wasn't sure if we would always want postgres as the metadata db.
If not ... then, we can make this more general.

(ditto for the blob storage)

Reply:
👍 this seems reasonable. we could add a comment that these params match if using the sample yamls